page-components/form-fields.md

```diff
@@ -63,9 +63,9 @@ For screen readers, make sure the tab focus order is correct. Consider the order
 
 #### Default
 
-* Border: 1 px #75787b (Grey)
-* Height: 27 px
-* Padding: 10px 3px
+* Border: 1 px #919395 (Grey 60)
+* Height: 35 px (includes border, padding, and text height)
```

Comment:
The current implementation is 35px, yes (the screenshot in this PR is out of date, I removed those padding changes)